src/applications/static-pages/situation-updates-banner/bannerContainer.jsx

```diff
@@ -0,0 +1,47 @@
+import React, { useEffect, useState } from 'react';
+import { useFeatureToggle } from '~/platform/utilities/feature-toggles';
+import SituationUpdateBanner from './situationUpdateBanner';
+import { getDataForPath } from './helpers/getDataForPath';
+
+export const BannerContainer = () => {
+  const {
+    TOGGLE_NAMES,
+    useToggleValue,
+    useToggleLoadingValue,
+  } = useFeatureToggle();
+
+  const alternativeBannersEnabled = useToggleValue(
+    TOGGLE_NAMES.bannerUseAlternativeBanners,
+  );
+  const isLoadingFeatureFlags = useToggleLoadingValue();
+
+  const [bannerState, setBannerState] = useState(null);
+
+  useEffect(() => {
+    let pathForSituationUpdate = window.location.pathname;
+    if (pathForSituationUpdate.endsWith('/')) {
+      pathForSituationUpdate = pathForSituationUpdate.slice(0, -1);
+    }
```

Comment:
Bit of a nit so feel free to ignore, but I think this looks better using replace regex
```suggestion
     pathForSituationUpdate = pathForSituationUpdate.replace(/\/$/, '');
```